test/apiTest.js

```diff
@@ -67,11 +67,68 @@ describe('DEL /cats/:id', function () {
         request(app)
             .delete('/cats/9')
             .set('Accept', "text/html; charset=utf-8")
-            .expect( 'Content-Type', "text/html; charset=utf-8")
+            .expect('Content-Type', "text/html; charset=utf-8")
             .expect(`cat id '9' not found in database`) 
             .expect(404, done)
     });
 });
 
 // why do we need to use .end() and err? 
 
+describe('POST /cats', function () {
+
+    const data = {
+        "name": "JimJam",
```

Comment:
the keys for this use double-quotes, is there any possibility that could be affecting the put/post requests? wild speculation from me

Reply:
Will test this

Reply:
No, unfortunately it is not